gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -389,8 +389,8 @@ gst_tensor_time_sync_buffer_from_collectpad (GstCollectPads * collect,
     data = (GstCollectData *) walk->data;
     pad = (GstTensorCollectPadData *) data;
```

Comment:
I think this line should be deleted too. Am I right? :)

Reply:
no, this tensorpaddata is used to handle sync mode now. (it has old gst-buffer for time sync mode)

Reply:
Oh, I see, I misread the {} range. Thanks for checking. :smiley: